WebRoot/css/zm.css

```diff
@@ -3478,10 +3523,10 @@ HTML>BODY .appt-selected .appt_allday_body
 
 /* not sure that these pseudoclasses will work in IE6 */
 .ZmAutocompleteListView TD:first-child {
-	padding-left:3px;
+	padding-left:15px;
```

Comment:
You can use `SkinWrapperPadding` variable which is `16px` and used throughout the site for maintaining even spacing. 

Thanks